tests/Extensions/TimedScopes.UnitTests/TaskExtensionsTests.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using Microsoft.Omex.Extensions.Abstractions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+
+namespace Microsoft.Omex.Extensions.TimedScopes.UnitTests
+{
+	[TestClass]
+	public class TaskExtensionsTests
+	{
+		[TestMethod]
+		public void WithTimedScopeWithTaskOfT_Succes() => TestSuccesfulExecution(s_wrapTaskOfTAction);
```

Comment:
Typo in `Success` throughout the method names

Reply:
I will resolve it in the follow-up pr https://github.com/microsoft/Omex/pull/168